junifer/data/parcellations.py

```diff
@@ -61,27 +65,28 @@
         "family": "Tian",
         "scale": scale,
         "magneticfield": "7T",
-        "space": "MNI6thgeneration",
+        "space": "MNI152NLin6Asym",
     }
     t_name = f"TianxS{scale}x3TxMNI6thgeneration"
     _available_parcellations[t_name] = {
         "family": "Tian",
         "scale": scale,
         "magneticfield": "3T",
-        "space": "MNI6thgeneration",
+        "space": "MNI152NLin6Asym",
     }
     t_name = f"TianxS{scale}x3TxMNInonlinear2009cAsym"
     _available_parcellations[t_name] = {
         "family": "Tian",
         "scale": scale,
         "magneticfield": "3T",
-        "space": "MNInonlinear2009cAsym",
+        "space": "MNI152NLin2009cAsym",
     }
 # Add AICHA parcellation info
 for version in (1, 2):
     _available_parcellations[f"AICHA_v{version}"] = {
         "family": "AICHA",
         "version": version,
+        "space": "MNI152Lin6Asym",
```

Comment:
We need confirmation that this is the exact space.

Reply:
To be exactly specific, this is what SPM 12 uses: `IXI549Space` ([ref](https://bids-specification.readthedocs.io/en/latest/appendices/coordinate-systems.html)). Now, on the MNI side, it's close to `MNI152Lin6Asym` ([ref](https://www.lead-dbs.org/about-the-mni-spaces/)). So, I can change it to `IXI549Space` if you agree.

Reply:
Is not what I meant. They provide the atlas for SPM12, and that table shows the default. 

From the paper:

Pre-processing was performed using [Statistical Parametric Mapping](https://www.sciencedirect.com/topics/neuroscience/statistical-parametric-mapping) subroutines (SPM5)

Later on:

normalized to the stereotaxic space of the Montreal Neurological Institute (MNI) template ([Ashburner and Friston, 2005]

But they don't specify which MNI space.


From the README of v2:

- "AICHA was projected in the MNI space as defined by the SPM12 template ". I guess this is the IXI549Space


But again, we don't know much about the v1 or the "guess".

Reply:
When you check https://www.gin.cnrs.fr/en/tools/aicha/ you would see they offer both v1 and v2 for SPM12 which we use. Now after that, it goes back to the reply I had earlier.

Reply:
New take:

1) Set it to `IXI549Space`
2) Raise a "warning" when this atlas is being used as we are not so sure about the space.
3) Contact the authors to clarify.

What do you think?

Reply:
Sounds good.